samples/https/nginx-app.yaml

```diff
@@ -33,7 +33,7 @@ spec:
           name: nginxconfigmap
       containers:
       - name: nginxhttps
-        image: ymqytw/nginxhttps:1.5
+        image: nginx:1.14.2
```

Comment:
I think we can actually remove this whole file, its not used in the docs and clearly super old  (see "ReplicationController")

Reply:
Get.